protocol/x/delaymsg/types/keys.go

```diff
@@ -1,9 +1,22 @@
 package types
 
+// Module name and store keys
 const (
 	// ModuleName defines the module name
 	ModuleName = "delaymsg"
 
 	// StoreKey defines the primary module store key
 	StoreKey = ModuleName
 )
+
+// State
+const (
+	// BlockMessageIdsPrefix is the prefix to retrieve all BlockMessageIds for a given block height.
+	BlockMessageIdsPrefix = "block_message_ids/"
```

Comment:
Nit: some of the prefixes string use plural form (example: here, `EpochStatsKeyPrefix` below). Most use singular form (like `delayed_message/` below). Could we maintain consistency?

Reply:
The name is based on the value ultimately stored at the location, it is not based on the key. So I believe it is already consistent as it is pluralized if the value is pluralized. Pluralization doesn't have to do with whether it is a collection of things or not